x-pack/plugins/monitoring/public/alerts/lib/alerts_toast.tsx

```diff
@@ -76,13 +76,13 @@ const showUnableToDisableWatcherClusterAlertsError = () => {
 
 const showDisabledWatcherClusterAlertsError = () => {
   Legacy.shims.toastNotifications.addWarning({
-    title: toMountPoint(
+    title: mountReactNode(
       <FormattedMessage
         id="xpack.monitoring.healthCheck.disabledWatches.title"
         defaultMessage="New alerts created"
       />
     ),
-    text: toMountPoint(
+    text: mountReactNode(
       <p>
```

Comment:
Not sure why we are wrapping the text in a `<p>` tag, we could remove it and check if it looks good visually.

Reply:
Any tips on getting these to fire for visual inspection? Just by recognizing the title, I guess I can probably use the "create alerts" dialog, but there's a bunch here I'm not sure I'd know what user flow would activate them.

Guess I can just mangle the code so they fire regardless of whatever conditions are supposed to feed them?

Reply:
![Screen_Shot_2021-12-14_at_14_04_28_and_Screen_Shot_2021-12-14_at_14_05_00_and_kibana_–_alerts_toast_tsx](https://user-images.githubusercontent.com/690/145936652-e25a20cf-4d4a-4a1e-b501-fa2c7be1a200.png)

Here's the `<p>` comparison. I personally like it without the extra whitespace so I think I'll switch this one to just an i18n message. 

I poked around a bit and it seems like the `<p>` should be included when there's another element like on https://github.com/elastic/kibana/blob/5b359a90b0892d15a92ec2cb7bfbd199e96e9b97/x-pack/plugins/apm/public/components/app/service_inventory/index.tsx#L127-L142